messages/pt.json

```diff
@@ -4,6 +4,9 @@
   "store/shipping-calculator.edit": "alterar",
   "store/shipping-calculator.estimate": "calcular",
   "store/shipping-calculator.viewDeliveryOptions": "Ver opções de entrega",
+  "store/shipping-calculator.allItemsUnavailable": "Nenhum dos produtos pode ser entregue nesse endereço.",
+  "store/shipping-calculator.unavailableItems": "Atenção a {numberOfUnavailableItems, plural, one {# produto que não pode ser entregue} other {# produtos que não podem ser entregues}} neste endereço.",
```

Comment:
```suggestion
  "store/shipping-calculator.unavailableItems": "Atenção, há {numberOfUnavailableItems, plural, one {# produto que não pode ser entregue} other {# produtos que não podem ser entregues}} neste endereço.",
```

Reply:
I think that what @augustocb wanted to say is "to pay attention TO the products", but he can say better.

Reply:
"Nenhum dos produtos pode**m** ser entregue nesse endereço".

"Podem" instead of "pode".

Reply:
In this case, I think it's also "entregues" instead of "entregue", right? @davicosta99

Reply:
All the messages:

**EN**
"This product can't be delivered to this address"
"All the products can't be delivered to this address."

"A product that can't be delivered to this address is highlighted."
"# products that can't be delivered to this address are highlighted."

**PT-BR**
"Esse produto não pode ser entregue nesse endereço."
"Nenhum dos produtos podem ser entregues nesse endereço."

"Um produto que não pode ser entregue nesse endereço está sinalizado."
"# produtos que não podem ser entregues nesse endereço estão sinalizados."

**ES**
"Este producto no se puede entregar en esta dirección."
"Ningún de los productos se puede entregar en esta dirección."

"Se encuentra resaltado un producto que no se puede entregar en esta dirección."
"Se encuentran marcados # productos que no se pueden entregar en esta dirección."

-----
Sorry for the confusion, I'll make sure to include these translations along with the UI in the future.

Reply:
Final fix (I hope):

"The product can't be delivered to this address"
"O produto não pode ser entregue nesse endereço."
"El producto no se puede entregar en esta dirección."